app/styles/helpers/_mixins.scss

```
@@ -21,3 +21,80 @@
         }
     }
 }
+
+@mixin d-gird($columns, $rows, $gap) {
+    display: grid;
+    grid-template-columns: $columns;
+    grid-template-rows: $rows;
+    grid-gap: $gap;
+}
+
+@mixin item {
+    .item__img {
+        position: relative;
+        flex-shrink: 0;
+
+        &:hover > .overlay {
+            opacity: 1;
+        }
+    }
+    .item__detail {
```

```suggestion
.product-item {
    &__img {
        position: relative;
        flex-shrink: 0;

        &:hover > .overlay {
            opacity: 1;
        }
    }
    &__detail {
```
Mixin là những thứ nho nhỏ dùng để khai báo các thuộc tính dễ dàng, ngắn gọn hơn thôi. Giống như các mixin d-flex, d-grid, text ở trên
Còn nguyên 1 cục product này thì tạo cho nó 1 class riêng nhé
Và bỏ nó vào 1 file riêng trong folder components. Folder components là nơi để chứa các component có tính tái sử dụng nhiều (breadcrumb, pagination, product, post, input, button,...)
VD: /scss/components/_product-item.scss